### PR TITLE
Jna550

### PR DESCRIPTION
**Pull request by kuznetsovmoci**

### CODE REVIEW

File: src/purejavahidapi/windows/CfgmgrLibrary.java [hunk 1]
```diff
@@ -32,10 +32,10 @@
 import com.sun.jna.Library;
 import com.sun.jna.Native;
 import com.sun.jna.win32.W32APIOptions;
-
+/*
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/CfgmgrLibrary.java]: Should this file be removed?

File: src/purejavahidapi/windows/HidDevice.java [hunk 2]
```diff
@@ -154,29 +161,35 @@ synchronized public int setOutputReport(byte reportID, byte[] data, int length)
 			throw new IllegalArgumentException("this device supports no output reports");
 		// In Windows writeFile() to HID device data has to be preceded with the report
 		// number, regardless
+		/*
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/HidDevice.java]: Remove this block

File: src/purejavahidapi/windows/Kernel32Library.java [hunk 3]
```diff
@@ -31,16 +31,23 @@
 
 import com.sun.jna.Native;
 import com.sun.jna.Pointer;
+//import com.sun.jna.platform.win32.WinBase;
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/Kernel32Library.java]: Remove these comments (34-36, 44-47)

File: src/purejavahidapi/windows/User32Library.java [hunk 4]
```diff
@@ -40,24 +40,26 @@
...
+//import purejavahidapi.windows.WinDef.HWND;
+//import purejavahidapi.windows.WinDef.LPARAM;
+//import purejavahidapi.windows.WinDef.LPVOID;
+//import purejavahidapi.windows.WinDef.LRESULT;
+//import purejavahidapi.windows.WinDef.POINT;
+//import purejavahidapi.windows.WinDef.PVOID;
+//import purejavahidapi.windows.WinDef.WPARAM;
+import com.sun.jna.platform.win32.WinDef.*;
+import com.sun.jna.platform.win32.WinNT.HANDLE;
+/*
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/User32Library.java]: Should this file be removed?

File: src/purejavahidapi/windows/WinDef.java [hunk 5]
```diff
@@ -32,15 +32,11 @@
...
 import purejavahidapi.windows.SetupApiLibrary.GUID;
 
-import com.sun.jna.FromNativeContext;
-import com.sun.jna.IntegerType;
-import com.sun.jna.Pointer;
-import com.sun.jna.PointerType;
-import com.sun.jna.Structure;
 import com.sun.jna.win32.StdCallLibrary.StdCallCallback;
-
+/*
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/WinDef.java]: Should this file be removed?

File: src/purejavahidapi/windows/WtsApi32Library.java [hunk 6]
```diff
@@ -29,16 +29,16 @@
...
 package purejavahidapi.windows;
 
-import purejavahidapi.windows.WinDef.HWND;
+import com.sun.jna.platform.win32.WinDef.HWND;
 
 import com.sun.jna.Native;
 import com.sun.jna.win32.StdCallLibrary;
 import com.sun.jna.win32.W32APIOptions;
-
+/*
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/WtsApi32Library.java]: Should this file be removed?

File: src/purejavahidapi/windows/WindowsBackend.java [hunk 7]
```diff
@@ -33,20 +33,34 @@
 import java.util.LinkedList;
 import java.util.List;
 
+//import com.sun.jna.platform.win32.SetupApi;
```
Comment (mitza-oci) [inline src/purejavahidapi/windows/WindowsBackend.java]: Remove comments in the import block

**Review by mitza-oci — CHANGES_REQUESTED** [review]

File: src/purejavahidapi/windows/CfgmgrLibrary.java (hunk 1 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/CfgmgrLibrary.java]: Yes - just waited for approval. Done now.

File: src/purejavahidapi/windows/HidDevice.java (hunk 2 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/HidDevice.java]: Done

File: src/purejavahidapi/windows/Kernel32Library.java (hunk 3 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/Kernel32Library.java]: Done

File: src/purejavahidapi/windows/User32Library.java (hunk 4 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/User32Library.java]: Done

File: src/purejavahidapi/windows/WinDef.java (hunk 5 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/WinDef.java]: Done

File: src/purejavahidapi/windows/WtsApi32Library.java (hunk 6 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/WtsApi32Library.java]: Done

File: src/purejavahidapi/windows/WindowsBackend.java (hunk 7 above)
Comment (kuznetsovmoci) [reply src/purejavahidapi/windows/WindowsBackend.java]: Done

**Review by mitza-oci — APPROVED** [review]

**Comment by mitza-oci** [comment]
JARs still need to get updated, or we need to handle that separately.

**Pull request merged**